sass/themes/blue/variables.scss

```diff
@@ -0,0 +1 @@
+$page-component-border-color: blue;
```

Comment:
please aware on naming convention. page-component.... was not necessary. page.... shall be nice